packages/aws-cdk-lib/aws-certificatemanager/lib/private-certificate.ts

```diff
@@ -28,6 +28,13 @@ export interface PrivateCertificateProps {
    * Private certificate authority (CA) that will be used to issue the certificate.
    */
   readonly certificateAuthority: acmpca.ICertificateAuthority;
+
+  /**
+   * Specifies the algorithm of the public and private key pair that your certificate uses to encrypt data.
+   *
```

Comment:
Would it be worth mentioning here that the value needs to match the `KeyAlgorithm` "family" of the Private Certificate Authority?

From CloudFormation:

> When you request a private PKI certificate signed by a CA from AWS Private CA, the specified signing algorithm family (RSA or ECDSA) must match the algorithm family of the CA's secret key.